### PR TITLE
migrations to add columns with enum type to the profile table

### PR DESCRIPTION
**Pull request by shriramshankar**

### CODE REVIEW

**Comment by shriramshankar** [comment]
Tested this in dev box and everything looks ok after deploy.

**Comment by shriramshankar** [comment]
@iamigo combined all the add column SQLs into a single one. I will create another PR for the writeup in "/docs/40-coredevelopment.html".

**Comment by shriramshankar** [comment]
@iamigo The build passed on this PR. Can you approve this, please?

File: migrations/20170811194929-add-profile-imc-fields.js [hunk 1]
```diff
@@ -6,87 +6,87 @@
...
+    return qi.sequelize.transaction(() => qi.describeTable(TBL)
     .then((attributes) => {
       attr = attributes;
-      if (!attr.hasOwnProperty('botAccess')) {
-        return qi.addColumn(TBL, 'botAccess', {
-          type: Sequelize.ENUM('r', 'rw'),
-          defaultValue: 'r',
-        });
-      } else {
+      if (attr.hasOwnProperty('botAccess')) {
```
Comment (iamigo) [inline migrations/20170811194929-add-profile-imc-fields.js]: missing "!"

File: migrations/20170811194929-add-profile-imc-fields.js [hunk 2]
```diff
@@ -6,87 +6,87 @@
...
+      });
     })
     .then(() => {
-      if (!attr.hasOwnProperty('eventAccess')) {
-        return qi.addColumn(TBL, 'eventAccess', {
-          type: Sequelize.ENUM('r', 'rw'),
-          defaultValue: 'r',
-        });
-      } else {
+      if (attr.hasOwnProperty('eventAccess')) {
```
Comment (iamigo) [inline migrations/20170811194929-add-profile-imc-fields.js]: missing "!"

File: migrations/20170811194929-add-profile-imc-fields.js [hunk 3]
```diff
@@ -6,87 +6,87 @@
...
+      });
     })
     .then(() => {
-      if (!attr.hasOwnProperty('roomAccess')) {
-        return qi.addColumn(TBL, 'roomAccess', {
-          type: Sequelize.ENUM('r', 'rw'),
-          defaultValue: 'rw',
-        });
-      } else {
+      if (attr.hasOwnProperty('roomAccess')) {
```
Comment (iamigo) [inline migrations/20170811194929-add-profile-imc-fields.js]: missing "!"

File: migrations/20170811194929-add-profile-imc-fields.js [hunk 4]
```diff
@@ -6,87 +6,87 @@
...
+      });
     })
     .then(() => {
-      if (!attr.hasOwnProperty('roomTypeAccess')) {
-        return qi.addColumn(TBL, 'roomTypeAccess', {
-          type: Sequelize.ENUM('r', 'rw'),
-          defaultValue: 'r',
-        });
-      } else {
+      if (attr.hasOwnProperty('roomTypeAccess')) {
```
Comment (iamigo) [inline migrations/20170811194929-add-profile-imc-fields.js]: missing "!"

File: migrations/20170811194929-add-profile-imc-fields.js (hunk 1 above)
Comment (shriramshankar) [inline migrations/20170811194929-add-profile-imc-fields.js]: I changed the if condition a bit. I am just returning true here.

Comment (iamigo) [inline migrations/20170811194929-add-profile-imc-fields.js]: Ohhh that's what I get for only reading the diff and not the whole file

**Pull request merged**